server/src/routes/traffic.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
```

Comment:
router names in plural

Reply:
@daniel-pupczyk-rmtcfm-com Ok.

Reply:
traffic don't have plural
https://simple.wiktionary.org/wiki/traffic

Reply:
@rafal-ksiazek-rmtcfm-com  you did @piotr-skowronski-rmtcfm-com job

Reply:
`traffic DOESN'T have plural form/version` 😇

Reply:
`traffic` have plural version but it is the same as singular and Im pretty sure that this `traffic` here was singular